microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,43 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+
+<!-- ====================================================================== -->
+
+## 1.0.1245.22
+  
+Release Date: June 15, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1245.22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1245.22)  
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 102.1245.22 or higher.

```

Comment:
```suggestion
For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 102.0.1245.22 or higher.
```

Reply:
@mikehoffms Can we fix the version number and release date?

Reply:
fixed version # 2x in https://github.com/MicrosoftDocs/edge-developer/pull/2011/files